.gitignore

```diff
@@ -24,3 +24,6 @@
 vendor/gems
 /public/packs
 /node_modules
+
+# Ignore test reports
+/test/reports/
```

Comment:
if it used tmp/test_reports, then they would not be in the way when searching for code in the test directory

Reply:
Good idea, done.